pkg/controller/kameletbinding/common.go

```diff
@@ -41,6 +42,10 @@ func createIntegrationFor(ctx context.Context, c client.Client, kameletbinding *
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: kameletbinding.Namespace,
 			Name:      kameletbinding.Name,
+			Labels: map[string]string{
+				kubernetes.CamelCreatorLabelKind: kameletbinding.Kind,
```

Comment:
I think this conflicts with something I've committed this morning (inheritance of annotations and labels). They should be added now.

Reply:
Sure, will do!